styles/style.css

```diff
@@ -24,24 +24,24 @@ figure {
 
 .app {
   display: flex;
-  flex-direction: column
+  flex-direction: column;
 
   min-height: 100vh;
 }
 
 .app__nav {
   background: #222;
-  color: rgb(220,220,230
+  color: rgb(220,220,230);
 
   padding: 1rem;
 }
 
 .app__content {
-  flex-grow: 1
+  flex-grow: 1;
 }
 
 .container {
-  max-width 60 rem;
+  max-width: 60 rem;
```

Comment:
We'll be reviewing units more tomorrow, but remove the space between `60` and `rem` to get a valid unit size and that should drastically update the styles of the page.